molecule/default/requirements.yml

```diff
@@ -1,4 +1,6 @@
 ---
 - src: elastic.elasticsearch
-  version: 5.5.1
+  version: 6.4.0
+- src: geerlingguy.kibana
+  version: 3.1.0
```

Comment:
https://github.com/entercloudsuite/ansible-elasticsearch/blob/87f6da17be147bf6964fa86bd9034289084257ee/molecule/default/requirements.yml#L4-L5

you must use [entercloudsuite.kibana](https://github.com/entercloudsuite/ansible-kibana) instead